modules/server_processes_manager/php/mriuploadserverprocess.class.inc

```diff
@@ -112,6 +112,13 @@ class MriUploadServerProcess extends AbstractServerProcess
             );
         }
 
+        // throw error if unable to locate insertion script
+        if (!file_exists($this->_mriCodePath . self::IMAGING_UPLOAD_FILE_PATH)) {
```

Comment:
I believe the check for the presence of the imaging upload script is not necessary. An exception will be thrown anyway if an attempt to start this process is made and the script does not exist.